hugo/content/contribute.md

```diff
@@ -96,15 +98,16 @@ Take a look at open [issues](https://github.com/mumble-voip/mumble/issues) and [
 
 #### 3rd party Interfaces
 
-Web interfaces, web-viewers, administration programs, CMS website modules – all neat things we would love to see! These could be written in any language supported by our Ice based or gRPC based RPC Interfaces. You could start your own project or link up with an {{< wiki "3rd Party Applications" "already existing one" />}}. To get an idea about how interaction with the Mumble server (Murmur) might look like you can take a look at the example scripts in our git repositories script folder.
+Web interfaces, web-viewers, administration programs, CMS website modules – all neat things we would love to see! These could be written in any language supported by our Ice based or gRPC based RPC Interfaces. You could start your own project or link up with an {{< wiki "3rd Party Applications" "already existing one" />}}. To get an idea about how interaction with the Mumble server might look like, you can take a look at the example scripts in our [GitHub repository script folder](https://github.com/mumble-voip/mumble/tree/master/scripts).
```

Comment:
Uhm I don't think that a lot of scripts there are about this purpose. What this text is referring to is probably https://github.com/mumble-voip/mumble-scripts, isn't it? :thinking:

Reply:
Ah, good to know.